src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -31,6 +31,11 @@
         <service id="Symfony\Component\Serializer\Mapping\ClassDiscriminatorResolverInterface" alias="serializer.mapping.class_discriminator_resolver" />
 
         <!-- Normalizer -->
+        <service id="serializer.normalizer.flatten_exception" class="Symfony\Component\Serializer\Normalizer\FlattenExceptionNormalizer">
+            <!-- Run before serializer.normalizer.object -->
+            <tag name="serializer.normalizer" priority="-915" />
```

Comment:
If I remove the priority the normalize() method is called. 
denormalize is called with our without the priority.

With this PR (the custom normalizer of this PR was not being called, that's why the trace_as_string is not being set). Result in xdebug:
<img width="588" alt="Captura de ecrã 2020-05-28, às 08 47 24" src="https://user-images.githubusercontent.com/74459/83119973-4cc36a80-a0c8-11ea-8d06-f89d9c714203.png">

(just did a throw new \InvalidArgumentException() in an handler to force failing)